src/tools/ceph-dencoder/CMakeLists.txt

```diff
@@ -58,7 +58,9 @@ if(WITH_RADOSGW)
     rgw_types.cc
     ${CMAKE_SOURCE_DIR}/src/rgw/rgw_dencoder.cc)
   target_include_directories(denc-mod-rgw
-    SYSTEM PRIVATE "${CMAKE_SOURCE_DIR}/src/rgw")
+    SYSTEM PRIVATE "${CMAKE_SOURCE_DIR}/src/rgw"
+    SYSTEM PRIVATE "${CMAKE_SOURCE_DIR}/src/rgw/store/rados"
```

Comment:
driver/rados (and if this compiles as-is, it didn't need that include directory)

Reply:
will be fixed in the next push